core/source/Http.mint

```diff
@@ -152,7 +158,20 @@ module Http {
     |> Http.send()
   */
   fun jsonBody (body : Object, request : Http.Request) : Http.Request {
-    { request | body = `JSON.stringify(#{body})` }
+    { request |
+      body = `JSON.stringify(#{body})`,
+      headers =
+        if (hasHeader("Content-Type", request)) {
+          request.headers
```

Comment:
```suggestion
          request
```

Reply:
I think it's fine, we are setting the headers here.

Reply:
Shouldn't the `request` be returned from here in order to allow chaining?

Reply:
@Sija The value of `headers` is the result of an `if` expression. In a simpler form, it's basically doing the following:

```mint
fun jsonBody (body : Object, request : Http.Request) : Http.Request {
  { request |
    body = `JSON.stringify(#{body})`,
    headers = if (/* header exists */) { request.headers } else { /* insert into headers */ }
  }
}
```

Reply:
Now I see it, sorry for the confusion 🙏